compiler/rustc_lexer/src/cursor.rs

```diff
@@ -61,8 +62,9 @@ impl<'a> Cursor<'a> {
     }
 
     /// Returns amount of already consumed symbols.
-    pub(crate) fn len_consumed(&self) -> usize {
-        self.initial_len - self.chars.as_str().len()
+    pub(crate) fn len_consumed(&self) -> u32 {
+        let len = self.initial_len - self.chars.as_str().len();
```

Comment:
Should we make `initial_len` `u32`?

Reply:
That wouldn't reduce the number of `as usize` or `as u32` needed.

Reply:
Yes, but I don't see why we need `initial_len` to be `usize` when the total len is `u16`, unless the `usize` for the computer is `u8` but otherwise it wouldn't happen right?

Reply:
We don't need it, but it would be more annoying to implement and not be a perf improvement anyway.